packages/router/Cargo.toml

```diff
@@ -38,22 +38,18 @@ serde = { version = "1", optional = true }
 serde_urlencoded = { version = "0.7.1", optional = true }
 
 [features]
-default = ["query"]
+default = ["query", "hot-reload"]
```

Comment:
I'm hesitant to put this in the default features right now. It gives access to change the html if anyone has access to ./_dioxus/hot_reload. For most cases this shouldn't be an issue, but for site hosting websites for example notify it could pose issues. It also almost definitely has worst performance right now (that should improve with subtree memorization).

Reply:
Whoops, got included on accident. Thanks for pointing it out.